src/odemis/gui/comp/legend.py

```diff
@@ -365,3 +364,101 @@ def on_paint(self, _):
             self._max_tick_width = max_width
             self.SetMinSize((self._max_tick_width + 14, -1))
             self.Parent.GetSizer().Layout()
+
+
+class RadioLegend(wx.Panel):
+    """
+    This class describes a legend containing radio buttons,
+    where each button represents a possible selection of a VA.
+    """
+
+    def __init__(self, parent, wid=-1, pos=(0, 0), size=wx.DefaultSize, style=wx.NO_BORDER):
+        wx.Panel.__init__(self, parent, wid, pos, size, style)
+
+        # get colors: if ar-viewport is selected, to avoid colors for bg and fg for txt will be taken from selection
```

Comment:
The comment is hard to read. I'd suggest something like:
`Store the colours now, as we'll need them whenever the radio button choices change, and by that time the parent (ARViewport) might be focused, which causes it to have a different background colour.`